metron-analytics/metron-profiler/pom.xml

```diff
@@ -339,6 +339,7 @@
                                 </excludes>
                             </artifactSet>
                             <transformers>
+                                <transformer implementation="org.atteo.classindex.ClassIndexTransformer"/>
```

Comment:
If I create a jar with Stellar functions, but no index file, then the function discovery will not work, correct?

If so, when using my own project outside of Metron that contains Stellar functions, I would need to add this transformer in my POM also, right?  If so, we should document how to do this somewhere, if you haven't already.